src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -175,7 +176,8 @@ object Config {
         stateSyncBloomFilterSize = syncConfig.getInt("state-sync-bloom-filter-size"),
         stateSyncPersistBatchSize = syncConfig.getInt("state-sync-persist-batch-size"),
         pivotBlockReScheduleInterval = syncConfig.getDuration("pivot-block-reschedule-interval").toMillis.millis,
-        maxPivotBlockAge = syncConfig.getInt("max-pivot-block-age")
+        maxPivotBlockAge = syncConfig.getInt("max-pivot-block-age"),
+        fastSyncMaxBatchRetries = syncConfig.getInt("fast-sync-max-batch-retries")
```

Comment:
This config entry `fast-sync-max-batch-retries` is missing from `application.conf`

Reply:
Correct, that's missing.